designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/table/ui/TableDimensions.kt

```diff
@@ -182,23 +231,38 @@ data class TableDimensions(
      * @param tableId The ID of the table.
      * @return The updated dimensions.
      */
-    fun resetWidth(tableId: String): TableDimensions {
+    fun resetWidth(
+        groupedTables: Boolean,
+        tableId: String,
+    ): TableDimensions {
+        val tableIdToUse = if (groupedTables) "GROUPED" else tableId
```

Comment:
I see "GROUPED" is used in a lot of places, it will be better is this is an constant variable